src/Workspaces/Core/Portable/Options/GlobalOptionService.cs

```diff
@@ -14,6 +14,8 @@
 using Microsoft.CodeAnalysis.Shared.Utilities;
 using Roslyn.Utilities;
 
+#pragma warning disable RS0030 // Do not use banned APIs (IGlobalOptionService)
```

Comment:
are you hoping to remove this later?

Reply:
No, the tests are actually testing the legacy APIs.